### DISCUSSION TITLE
Everyone fixing the same issue

### DISCUSSION THREAD

**Discussion by ragaumon** [gitextensions/gitextensions#10328] [General]
Everyone (except Arturas) was fixing "prompt (delete, save, cancel) when draft has no changes"
Why it happens, my ideas:
1. Paulius did change with another task, and changes went into the dev branch. It created some bugs related to this task.
2. My changes were too slow, and everyone who find this bug tried to fix it.

**Comment by gerhardol** [comment]
Off topic for GE
